whois-api/src/main/java/net/ripe/db/whois/api/elasticsearch/ElasticIndexService.java

```diff
@@ -131,14 +131,20 @@ protected void deleteAll() throws IOException {
         client.deleteByQuery(request, RequestOptions.DEFAULT);
     }
 
+    protected void refreshIndex(){
+        try {
+            client.indices().refresh(new RefreshRequest(whoisAliasIndex), RequestOptions.DEFAULT);
```

Comment:
Did you check the performance impact for this ? https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-refresh.html. I think issue is we forgot to set the query in countRequest if  that does not solve the problem I suggest instead of refreshing entries every 60 seconds may be have just a count compare once in 2-3 hours ?

Reply:
Discussed via Mattermost, we will create a job that perform the check each hour

Reply:
@MiguelAHM tested that the performance impact of a refresh is minimal (less than 0.5s)